stubs/pywin32/win32/win32gui.pyi

```diff
@@ -42,6 +48,7 @@ def PostMessage(
 ) -> None: ...
 def PostThreadMessage(threadId, message, wparam, lparam) -> None: ...
 def ReplyMessage(result): ...
+def ResetDC(__hdc: int, __devmode: _win32typing.PyDEVMODE | _win32typing.PyDEVMODEW) -> int: ...
```

Comment:
I'll just trust you on this one after what happened in the REPL with the last one!